### PR TITLE
adding David Lange's comments built on rebase of ValidationDoubleCountin...

### PR DESCRIPTION
**Pull request by inugent**
...gFix fixing missing files
Replaces pull 8676

### CODE REVIEW

**Comment by inugent** [comment]
Note: missing files refers to files missing by git add when setting up this branch (they were in the original pull request 8676).

**Comment by deguio** [comment]
in general there is no need to open a new PR. pushing to the same branch updates the existing one.

File: Validation/EventGenerator/plugins/BPhysicsValidation.cc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+BPhysicsValidation::~BPhysicsValidation(){}
+
+void BPhysicsValidation::dqmBeginRun(const edm::Run& r, const edm::EventSetup& c) {}
+
+void BPhysicsValidation::bookHistograms(DQMStore::IBooker &i, edm::Run const &, edm::EventSetup const &){
+  DQMHelper dqm(&i); i.setCurrentFolder("Generator/BPhysics");
+  Nobj   = dqm.book1dHisto("N"+name, "N"+name, 1, 0., 1,"bin","Number of "+name);
+  particle.Configure(i);
+  for(unsigned int j=0;j<daughters.size();j++){std::cout << j << std::endl; daughters[j].Configure(i);}
```
Comment (deguio) [inline Validation/EventGenerator/plugins/BPhysicsValidation.cc]: please remove the cout

**Comment by deguio** [comment]
+1

**Comment by bendavid** [comment]
-1
This can be closed in favour of the 76x version
https://github.com/cms-sw/cmssw/pull/9765

**Comment by inugent** [comment]
replaced by #9765

**Pull request closed without merging**